local_rl_loop.py

```diff
@@ -79,6 +80,10 @@ def rl_loop():
             holdout_pct=100,
             readouts=10)
 
+        print("See sgf files here?")
```

Comment:
Not sure I understand this comment.

Reply:
these are 'integration tests' with big scare quotes.  If you're running this and you want to see that everything still works, you should see some files with nonzero size.